java/src/jmri/jmrit/roster/swing/RenameRosterGroupAction.java

```diff
@@ -81,18 +81,19 @@ public void actionPerformed(ActionEvent event) {
         }
 
         String entry = (String) JOptionPane.showInputDialog(_who,
-                "<html><b>Rename roster group</b><br>Enter the new name for roster group \"" + group + "\".</html>",
-                "Rename Roster Group " + group,
+                Bundle.getMessage("RenameRosterGroupNewName", group),
+                Bundle.getMessage("RenameRosterGroupTitle", group),
                 JOptionPane.INFORMATION_MESSAGE,
                 null,
                 null,
                 null);
-        if (entry == null || entry.equals(Roster.ALLENTRIES)) {
+        entry = entry.trim(); // remove white space around name, also prevent "Space" as a Group name
```

Comment:
Same concerns as earlier, but also note that if entry is null (which is valid in most of the JMRI code), this trim will cause an NPE.